connect-inject/connect_sidecar.go

```diff
@@ -0,0 +1,46 @@
+package connectinject
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"strings"
+)
+
+func (h *Handler) connectSidecar(pod *corev1.Pod) corev1.Container {
+	command := []string{
+		"consul-k8s",
+		"connect-sidecar",
```

Comment:
The command is generically named "connect-sidecar" (i.e. rather than
re-registerer) to make room for more functionality in the future, e.g.
renewing tokens.

We could change that later I suppose, however folks might have tooling built around the name. I don't mind either way.

Reply:
Ah that's right! How about `consul-agent-sidecar`?

Reply:
That's a bit misleading as consul-agent might make folks think of an actual `consul agent` running in there.